content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Acessando serviços em execução em clusters
+content_type: task
+weight: 140
+---
+
+<!-- overview -->
+Esta página mostra como se conectar aos serviços em execução no cluster Kubernetes.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Acessando serviços em execução no cluster
+
+No Kubernetes, [nós](/pt-br/docs/concepts/architecture/nodes/), [Pods](/docs/concepts/workloads/pods/) e [serviços](/docs/concepts/services-networking/service/) têm todos seus próprios IPs. Em muitos casos, os IPs dos nós, dos Pods e alguns dos IPs de serviço em um cluster não serão
+roteáveis, portanto, não estarão acessíveis a partir de uma máquina fora do cluster, como seu computador.
+
+### Maneiras de se conectar
+
+Você tem várias opções para se conectar a nós, pods e serviços de fora do cluster:
+
+  - Acesse serviços através de IPs públicos.
+    - Use um serviço com tipo `NodePort` ou `LoadBalancer` para tornar o serviço acessível fora do cluster. Consulte a documentação de [serviços](/docs/concepts/services-networking/service/) e
+[kubectl expose](/docs/reference/generated/kubectl/kubectl-commands/#expose).
+    - Dependendo do ambiente do cluster, isso pode expor o serviço apenas para a rede corporativa, ou pode expô-lo para a Internet. Pense se o serviço que está sendo exposto é seguro. Ele faz sua própria autenticação?
+    - Coloque Pods atrás de serviços. Para acessar um Pod específico de um conjunto de réplicas, como para depurar, coloque um rótulo exclusivo no Pod e crie um novo serviço que selecione este rótulo.
```

Comment:
"coloque uma label exclusiva no Pod e ... selecione esta label." instead of "coloque um rótulo exclusivo no Pod e ... selecione este rótulo.". In this context, the "label" is a key-value pair.